images/filebeat/tests/values.yaml

```diff
@@ -0,0 +1,80 @@
+daemonset:
+  filebeatConfig:
+    filebeat.yml: |
+      filebeat.inputs:
+      - type: container
+        paths:
+          - /var/log/containers/*.log
+        processors:
+        - add_kubernetes_metadata:
+            host: ${NODE_NAME}
+            matchers:
+            - logs_path:
+                logs_path: "/var/log/containers/"
+      output.file:
+          path: "/tmp/"
+          filename: foobar
+      setup.ilm.enabled: false
+      setup.template.name: "filebeat"
+      setup.template.pattern: "filebeat-oss-*"
+  resources:
+    limits:
+      # Should avoid OOM (Error 137) when running goss tests into the pod
+      memory: "300Mi"
```

Comment:
I'd recommend setting requests here as well so this doesn't flake. once this lands on HEAD it's going to be a very noisy environment